lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -195,22 +199,28 @@ def _check_summing_mismatch(track, sampling_width):
             else:
                 return RuntimeWarning(wrong_kymo_warning)
     except IndexError:
-        return RuntimeWarning(wrong_kymo_warning)
+        raise ValueError(wrong_kymo_warning)
 
 
-def import_kymotrackgroup_from_csv(filename, kymo, channel, delimiter=";"):
-    """Import a KymoTrackGroup from a csv file.
+def load_tracks(filename, kymo, channel, delimiter=";"):
+    """Loads a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` from a csv file.
 
     The file format contains a series of columns as follows:
     track index, time (pixels), coordinate (pixels), time (optional), coordinate (optional),
     sampled_counts (optional), minimum length
 
+    .. note::
+
+        If you cropped the kymograph prior to tracking, you will have to make sure that you crop
+        the kymograph you supply to this function in the same way.
+
     Parameters
     ----------
     filename : str | os.PathLike
         filename to import from.
     kymo : Kymo
-        kymograph instance that the CSV data was tracked from.
+        Kymograph instance that the CSV data was tracked from. This kymograph has to be the same
+        as the one used to create the tracks.
```

Comment:
I think `instance` might not be the best word here since more literal-minded users might interpret it as having to be the same object (think identity, not equality). I think the new phrase is clearer and mostly obsoletes the previous one. I would also mention stuff like ROI also matters and refer users to read the `note` above.

Reply:
Good point! I rephrased it to:

```
Kymograph that the CSV data was tracked from. This kymograph has to be the one used to
create the tracks (this includes any processing). See the note above for more information.
```